app/organisation/rest.py

```diff
@@ -353,3 +356,24 @@ def _send_notification(template_id, recipient, personalisation):
     _send_notification(
         current_app.config[signer_template_id], organisation.agreement_signed_by.email_address, personalisation
     )
+
+
+@organisation_blueprint.route("/<uuid:organisation_id>/organisation_permission/<string:permission>", methods=["POST"])
```

Comment:
This route could probably just be `/permission/` to remove some redundancy - what do you think?